app/Http/Controllers/EventAttendeesController.php

```diff
@@ -662,7 +662,7 @@ public function postCancelAttendee(Request $request, $event_id, $attendee_id)
                 $message->to($attendee->email, $attendee->full_name)
                         ->from(config('attendize.outgoing_email_noreply'), $attendee->event->organiser->name)
                         ->replyTo($attendee->event->organiser->email, $attendee->event->organiser->name)
-                        ->subject('You\'r ticket has been cancelled');
+                        ->subject('Your ticket has been cancelled');
```

Comment:
I am not extremely experienced with Git, and will look up how to do that in the future.